.github/workflows/ci.yaml

```diff
@@ -25,6 +25,15 @@ jobs:
         run: |
           make lint
           scripts/check-sync-tidy.sh
-
+      - name: Lint scripts
```

Comment:
the script uses library versions that are only available in go1.21, anyway that is the only Go version where its worth testing it because we will execute it from local machine